charts/kubetail/Chart.yaml

```diff
@@ -8,7 +8,7 @@ keywords:
 - private
 - realtime
 type: application
-version: 0.2.8
+version: 0.3.0
```

Comment:
If this PR won't include rbac modifications then I think it's fine to bump to 0.2.9 if the other changes are backwards compatible.

Reply:
Normally, I bump the minor version because of new features.

Reply:
Ok, bumping minor makes sense